javascript/react-verbs/src/verbs/Convert/Convert.tsx

```diff
@@ -31,105 +31,100 @@ import { getColumnType } from '../shared/TypingFunction/TypingFunction.js'
 /**
  * Provides inputs for a Convert step.
  */
-export const Convert: React.FC<StepComponentProps> = memo(function Convert({
-	step,
-	store,
-	table,
-	onChange,
-	input,
-}) {
-	const internal = useMemo(() => step as ConvertStep, [step])
-	const tbl = useLoadTable(
-		input || internal.input[NodeInput.Source]?.node,
-		table,
-		store,
-	)
-	const [inputColumnDate, setInputColumnDate] = useState<boolean>()
+export const Convert: React.FC<StepComponentProps<ConvertArgs>> = memo(
+	function Convert({ step, store, table, onChange, input }) {
+		const tbl = useLoadTable(
```

Comment:
We could probably abstract all of these with an HOC that injects the resolved table?

Reply:
I'd rather use a hook that took a props object to streamline this. I'm personally not a fan of HOCs

Reply:
Eh, a HOC is cleaning this up pretty nicely